src/main/java/com/google/devtools/build/lib/remote/RemoteExecutionService.java

```diff
@@ -258,7 +259,7 @@ static Command buildCommand(
   public static class RemoteAction {
     private final Spawn spawn;
     private final SpawnExecutionContext spawnExecutionContext;
-    private final RemoteActionExecutionContext remoteActionExecutionContext;
+    private final RemoteActionExecutionContext context;
```

Comment:
Is it a good idea to rename this, given that there are two different context objects?